packages/x-components/src/components/modals/__tests__/x-modal.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import { getDataTestSelector, installNewXPlugin } from '../../../__tests__/utils';
+import XModal from '../x-modal.vue';
+import { PropsWithType } from '../../../utils/types';
+import { XEventsTypes } from '../../../wiring/events.types';
+
+/**
+ * Mounts a {@link XModal} component with the provided options and offers an API to easily
+ * test it.
+ *
+ * @param options - The options to render the component with.
+ * @returns An API to test the component.
+ */
+function mountXModal({ template = '<XModal/>' }: MountXModalOptions = {}): MountXModalAPI {
```

Comment:
`mount` or `render`? I don't mind changing to `mount`, but I think we have set it to `render` in our own naming convention 😅 

Same with the `options` and `API` names

```suggestion
function renderXModal({ template = '<XModal/>' }: RenderXModalOptions = {}): RenderXModalAPI {
```